internal/core/domain/wallet_repository.go

```diff
@@ -51,8 +51,8 @@ type WalletRepository interface {
 	// UnlockWallet attempts to update the status of the Wallet to "unlocked".
 	// Generates a WalletUnlocked event if successfull.
 	UnlockWallet(ctx context.Context, password string) error
-	// LockkWallet updates the status of the Wallet to "locked".
-	// Generates a WalletLocked event if successfull.
+	// LockWallet updates the status of the Wallet to "locked".
+	//Generates a WalletLocked event if successfull.
```

Comment:
Revert changes to this line:
```suggestion
	// Generates a WalletLocked event if successfull.
```

Reply:
Done